### ISSUE TITLE
Opting-out during play+rate affects ratings

### ISSUE THREAD

**Issue by Yngwarr** [ludumdare/ludumdare#1739]
![capture](https://user-images.githubusercontent.com/1721439/44225951-fb7b0d00-a1b8-11e8-90d1-175530037bbb.PNG)

I had 14 ratings for humor when I decided to opt-out from this category, but it wasn't excluded from 'ratings recieved' metric.

**Comment by local-minimum** [comment]
How quickly after is the screenshot? It may take a while to update, but it could well be that we haven't taken that into account.

**Comment by mikekasprzak** [comment]
Yeah at 3000+ games it might take 30-45 minutes for the change to
propagate.

**Comment by Yngwarr** [comment]
Sorry for a late response, it was about 5 minutes after my change but I'm convinced that it is a miscalculation because 20.5 is a sum of all my ratings, including humor, divided mistakenly by 4.

**Comment by Aurel300** [comment]
Same happened to us, it's not great that opting out seems to penalise you in the karma system.
